docs/modules/ROOT/pages/how-tos/restore_postgresql_burp.adoc

```diff
@@ -0,0 +1,56 @@
+= Restore PostgreSQL with BURP
+
+== Backup
+
+Dump destination directory: `/var/lib/postgresql-backup`
+
+The postgresql-pre-backup script uses https://www.postgresql.org/docs/current/app-pgdump.html[pg_dump]
+to export all PostgreSQL databases in a consistent data format before every BURP run, usually once a day.
+The destination files are named after the database name.
+
+The default dump format used is "custom", it may be overriden by DUMP_FORMAT in `/etc/default/postgresql-pre-backup`
+
+Example file list:
+
+[source,bash]
+--
+confluence.dump
+postgres.dump
+--
+
+== Restore
+
+Backups can be re-imported with the pg_restore command. The following commands should be seen as examples.
+An actual restore procedure depends on the situation at hand (partial restore after accidental data removal or a full restore after catastrophic hardware failure).
+
+NOTE: This will only restore data, but not roles. Roles are supposed to be configured via puppet/hieradata.
+
+. Restore from BURP (see xref:how-tos/restore_from_backup.adoc[Restore Files from Backup] for details and more options):
++
+[source,bash]
+--
+$ sudo burp -a r -d /tmp/restore -r ^/var/lib/postgresql-backup/
+$ sudo chown -R postgres:postgres /tmp/restore
+--
+
+NOTE: If the service runs on an encrypted disk, ensure you use a restore location
+that is on an encrypted disk as well.
```

Comment:
```suggestion
IMPORTANT: If the service runs on an encrypted disk, ensure you use a restore location that is on an encrypted disk as well.
```

Didn't see the "NOTE" box, so IMPORTANT is better perhaps.

Also no newline after 80 chars